action.yml

```diff
@@ -42,16 +42,16 @@ runs:
       path: summary
   # output true if any state in inputs.terraform-directory changed
   - id: summary
-    if: steps.check.outputs.skip == 'false'
+    if: (success() || failure()) && steps.check.outputs.skip == 'false'
     shell: bash
-    run: (cat summary/terraform.*.summary || echo "No summary files found!" && exit 1) | grep 'true' || echo "has_changes=false" >> $GITHUB_OUTPUT
+    run: (cat summary/terraform.*.summary || true) | grep 'true' || echo "has_changes=false" >> $GITHUB_OUTPUT
```

Comment:
the main change here is that it just ignores the error from non-existent artifact files.